lib/src/over_react_redux/redux_multi_provider.dart

```diff
@@ -68,7 +67,7 @@ class ReduxMultiProviderComponent
   get propTypes => {
         keyForProp((p) => p.storesByContext): (props, info) {
           final storesByContext = props.storesByContext;
-          if (storesByContext != null && storesByContext.isEmpty) {
```

Comment:
Looks like there's a failing test:
```
❌ [Chrome, Dart2Js] test/over_react_redux_test.dart: ReduxMultiProvider works as expected when storesByContext is null (failed)
  Expected: emits the propType warning some element contains 'RequiredPropError: Prop ReduxMultiProviderProps.storesByContext is required.'
    Actual: <Closure: () => TestJacket<Component> from: () => {
                        let t9;
                        return jacket$.mount(react.Component, redux_multi_provider.ReduxMultiProvider().call((t9 = dom_components.Dom.div(), (() => {
                          t9.addTestId("content");
                          return t9;
                        })()).call("foo")));
                      }>
     Which: has propType warning with value [
              'Warning: Failed prop type: Expected a value of type \'Map<Context<dynamic>, Store<dynamic>>\', but got one of type \'Null\''
            ]
```

That test should probably just be removed, since when we add required prop validation it'll fail for that case when invoking the builder.

Reply:
Removed the test in https://github.com/Workiva/over_react/pull/855/commits/136eb834d6cc04b101a76857716aa18ac9e04702